src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -633,27 +627,20 @@ GUID CascadiaSettings::_GetProfileForArgs(const NewTerminalArgs& newTerminalArgs
 //   If omitted, instead return the default profile's GUID
 // Return Value:
 // - the Nth profile's GUID, or the default profile's GUID
-GUID CascadiaSettings::_GetProfileForIndex(std::optional<int> index) const
+std::optional<GUID> CascadiaSettings::_GetProfileGuidByIndex(std::optional<int> index) const
 {
-    GUID profileGuid;
     if (index)
     {
-        const auto realIndex = index.value();
+        const auto realIndex{ index.value() };
         // If we don't have that many profiles, then do nothing.

```

Comment:
You reversed the logic but forgot to update the comment here.

Reply:
The comment is durable even if the logic checks for inclusivity instead of exclusivity ;P